setup.py

```diff
@@ -26,7 +26,7 @@
         "hvac>=0.7.0,<0.8.0",
         "ldap3>=2.9.1,<2.10.0",
         "anymarkup>=0.7.0,<0.9.0",
-        "python-gitlab>=1.11.0,<1.12.0",
+        "python-gitlab~=4.6",
```

Comment:
we may need to fix some breaking changes for this 5 years version upgrade https://python-gitlab.readthedocs.io/en/stable/changelog.html

Reply:
#4467